tests/periph_uart_nonblocking/app.config

```diff
@@ -1,4 +1,4 @@
-CONFIG_MOD_TEST_UTILS_INTERACTIVE_SYNC=y
-CONFIG_MOD_PERIPH_UART=y
-CONFIG_MOD_PERIPH_UART_NONBLOCKING=y
-CONFIG_MOD_XTIMER=y
+CONFIG_MODULE_TEST_UTILS_INTERACTIVE_SYNC=y
+CONFIG_MODULE_PERIPH_UART=y
+CONFIG_MODULE_PERIPH_UART_NONBLOCKING=y
+CONFIG_MODULE_XTIMER=y
```

Comment:
Seems like some squashing is needed, some commits have renamings of:

```diff
-CONFIG_MOD_TEST_UTILS_INTERACTIVE_SYNC=y
-CONFIG_MOD_PERIPH_UART=y
-CONFIG_MOD_PERIPH_UART_NONBLOCKING=y
-CONFIG_MOD_XTIMER=y
+CONFIG_MODULE_TEST_UTILS_INTERACTIVE_SYNC=y
+CONFIG_MODULE_PERIPH_UART=y
+CONFIG_MODULE_PERIPH_UART_NONBLOCKING=y
+CONFIG_MODULE_XTIMER=y
```

Reply:
Oh, sorry about that, poor rebasing and adaption to the new perfixes. I'll fix this